ability_item_usage_hoodwink.lua

```diff
@@ -92,6 +93,7 @@ local Consider = {}
 local CanCast = { utility.NCanCast, utility.NCanCast, utility.NCanCast, utility.UCanCast, utility.UCanCast,
 	utility.UCanCast }
 local enemyDisabled = utility.enemyDisabled
+-- local hoodwink_bushwhack_radius = 265
```

Comment:
Is this variable still in use? If not, can it be removed?

Reply:
It is now in use anymore and I`ll remove it.